duckduckgo/iOS
# Add new share icon

Reviewer: Francesco
Asana: https://app.asana.com/0/276208855376242/360339736346103

**Description**:
Updates sharing icon to a "logo" as share sheet is populated by many external applications

**Steps to test this PR**:
1. When browsing press the share button
2. Note that the "Add to bookmarks" icon is now our logo (was previously a generic bookmark icon)

###### Reviewer Checklist:


###### PR DRI Checklist:
